services/QuillLMS/spec/mailers/user_mailer_spec.rb

```diff
@@ -169,6 +169,47 @@
     end
   end
 
+  describe 'teacher_requested_verification_email' do
+    let(:teacher) { create(:teacher) }
+    let!(:admin) { create(:admin) }
+    let(:message) { 'Some message' }
+    let(:school) { create(:school) }
+    let!(:schools_user) { create(:schools_users, user: teacher, school:) }
+    let(:mail) { described_class.teacher_requested_verification_email(teacher, admin, message) }
+
+    before { teacher.reload }
```

Comment:
I'm totally willing to believe that this is a necessary line, but I can't imagine what `teacher.reload` would do in a `before` block since there haven't been any changes yet when this is triggered.  So are the tests actually failing without this?  (In an ideal world, I'd say figure out why and fix it, but it's not a big enough deal to be worth you spending a couple of hours on, I don't think.)

Reply:
It seems like it's actually forcing `teacher.school` to work correctly -- for some reason, even though `schools_user` is instantiated with a `let!`, teacher.school returns undefined otherwise.